src/scss/greenhouse.io.scss

```diff
@@ -222,10 +221,18 @@ a {
 #application_form {
 
     @include min-mobile-xl {
+        width: 568px;
         max-width: 568px;
```

Comment:
is the combination of width/max-width needed when they are the same value?

Reply:
If I recall correctly, some mobile devices have issues with setting only width or setting only max-width